src/main/features/dashboard/index.ts

```diff
@@ -64,5 +65,13 @@ export class DashboardFeature {
       }))
 
     app.use('/', RouterFinder.findAll(path.join(__dirname, 'routes')))
+    app.use((err, req, res, next) => {
+      if (err.statusCode === 500) {
```

Comment:
It's not a 500 until it's processed by `app.ts`
You could check that `err.stack.split('\n')[0].startsWith('Template render error')`